docs/kernel-policy.md

```diff
@@ -10,7 +10,8 @@ per [Firecracker’s release policy](../docs/RELEASE_POLICY.md)) using a
 combination of:
 
 - host linux kernel versions 5.10, and 6.1;
-- guest linux kernel versions 4.14, and 5.10.
+- guest linux kernel versions 5.10 and 6.10. Guest linux kernels 4.14 are now
```

Comment:
```suggestion
- guest linux kernel versions 5.10 and 6.1. Guest linux kernels 4.14 are now
```

I would maybe also drop the "now" here (maybe say "since 1.9" if we want to have some temporal information)